### DISCUSSION TITLE
Images are not displayed

### DISCUSSION THREAD

**Discussion by Sibiriak** [Zettlr/Zettlr#4140] [Q&A] [answered]
Probably a noob question but anyway...

I've installed Zettlr trying to move from Obsidian. Everything is okay except the fact that no attached images are displayed. For example, if the image link is ![[pasted_image.jpg]]  - it shows nothing, just the plain text. Is there any way in Zettlr to use image links like this?

**Comment by wheelercj** [comment]
Hello! Zettlr uses markdown-style image links, so like this: `![](pasted_image.jpg)`, or this: `![optional alt text](pasted_image.jpg)`. I wonder if someone in the community already has a script for converting from the Obsidian format to the markdown format? If not, I don't think it would take too long to make a Python one if it would be helpful.

**Comment by Sibiriak** [comment]
Well, I can write such a python script for my personal purposes but was wondering if  it's possible  to do by built-in tools. Anyway, it'd be hard to use such conversations all the time between accessing from Obsidian and Zettlr both.

**Comment by exanrauzer** [comment, marked as answer]
Isn't both types of link handled by Obsidian? If so, you can just convert them without a loss, and establish same format for Zettlr and Obsidian.

https://github.com/ozntel/obsidian-link-converter
